projects/admin/src/app/record/detail-view/document-detail-view/document-detail-view.component.html

```diff
@@ -0,0 +1,262 @@
+<!--
+  RERO ILS UI
+   Copyright (C) 2019 RERO
+  
+   This program is free software: you can redistribute it and/or modify
+   it under the terms of the GNU Affero General Public License as published by
+   the Free Software Foundation, version 3 of the License.
+  
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+   GNU Affero General Public License for more details.
+  
+   You should have received a copy of the GNU Affero General Public License
+   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+-->
+<ng-container *ngIf="record">
+  <header class="row">
+    <h1>{{ record.metadata.title }}</h1>
+  </header>
+  <section>
+  <dl class="row mb-0">
+    <!-- TYPE -->
+    <dt class="col-sm-3 offset-sm-2 offset-md-0">
+      {{ 'Type' | translate }}:
+    </dt>
+    <dd class="col-sm-7 col-md-8 mb-0">
+      {{ record.metadata.type | translate | titlecase }}
+    </dd>
+
+    <!-- AUTHORS -->
+    <ng-container *ngIf="authors as docAuthors">
+      <ng-container *ngIf="docAuthors.length > 0">
+      <dt class="col-sm-3 offset-sm-2 offset-md-0">
+        <ng-container *ngIf="docAuthors < 2; else authors">
+          {{ 'Author' | translate }}:
+        </ng-container>
+        <ng-template #authors>
+          {{ 'Authors' | translate }}:
+        </ng-template>
+      </dt>
+      <dd class="col-sm-7 col-md-8 mb-0">
+        <span *ngFor="let author of docAuthors; let isLast=last">
+          <span *ngIf="author.pid; else authorOnly">
+            <a [routerLink]="['/records', 'persons', 'detail', author.pid]">{{ author.name }}</a>{{ isLast ? '' : ', ' }}
+          </span>
+          <ng-template #authorOnly>
+            {{ author.name }}{{ isLast ? '' : '; ' }}
+          </ng-template>
+        </span>
+      </dd>
+      </ng-container>
+    </ng-container>
+
+    <!-- EDITION STATEMENT -->
+    <ng-container *ngIf="editionsStatement as editions">
+      <ng-container *ngIf="editions.length > 0">
```

Comment:
Cosmetic : Not specialist but it should be condensed into only one IF statement ?

Reply:
This is an optimization to avoid generating data a second time.